frontend/src/metabase/visualizations/components/ScalarValue.styled.tsx

```diff
@@ -11,3 +20,34 @@ export const ScalarRoot = styled.div`
   width: 100%;
   height: 100%;
 `;
+
+export const ScalarValueWrapper = styled.h1<ScalarValueProps>`
+  cursor: pointer;
+  &:hover {
+    color: ${color("brand")};
+  }
+
+  ${({ isDashboard, gridSize }) => {
+    if (!isDashboard || !gridSize) {
+      return undefined;
+    }
+
+    const { width } = gridSize;
+    let fontSize = "1.8rem";
+    let breakpointFontSize = "2.8rem";
+    if (width >= 9) {
+      fontSize = "3.8rem";
```

Comment:
Would it be easier to hit the right scale if it could be defined by the dashboard maker?

How does it handle numbers that are 2 long? What if the number is 5 long?